helpers/constants.js

```diff
@@ -4,6 +4,25 @@ exports.constants = {
 		email: "a0k03lf@gmail.com"
 	},
 	confirmEmails: {
-		from : "a0k03lf@gmail.com"
+		from: "a0k03lf@gmail.com"
+	},
+	userTypes: {
+		RECRUITER: "RECRUITER",
+		JOBSEEKER: "JOBSEEKER"
+	},
+	experienceType: {
+		FRESHER: 'FRESHER',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		FRESHER: "FRESHER",
```